assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -57,10 +73,83 @@ describe( 'modules/analytics accounts', () => {
 	afterEach( () => {
 		unsubscribeFromAll( registry );
 		apiFetchSpy.mockRestore();
+		windowLocationSpy.mockRestore();
 	} );
 
 	describe( 'actions', () => {
+		describe( 'createAccount', () => {
+			it( 'creates an account ticket and redirects to the Terms of Service', async () => {
+				fetch
+					.doMockIf(
+						/^\/google-site-kit\/v1\/modules\/analytics\/data\/create-account-ticket/
+					)
+					.mockResponse(
+						JSON.stringify( fixtures.accountTicket ),
+						{ status: 200 }
+					);
+
+				muteConsole( 'error' );
+
+				registry.dispatch( STORE_NAME ).createAccount( { accountName, propertyName, profileName, timezone } );
+				await subscribeUntil( registry,
+					() => (
+						registry.select( STORE_NAME ).isDoingCreateAccount() !== true
+					),
+				);
+
+				// Ensure the proper body parameters were sent.
+				expect( JSON.parse( fetch.mock.calls[ 0 ][ 1 ].body ).data ).toMatchObject(
+					{ accountName, propertyName, profileName, timezone }
+				);
+
+				expect( redirect ).toEqual( 'https://analytics.google.com/analytics/web/?provisioningSignup=false#management/TermsOfService/?api.accountTicketId=abc123' );
```

Comment:
Maybe use template literals to get the `abc123` from the `fixtures.accountTicket` object instead? Right now it's not clear where this value is looking from in the test.

Reply:
Good suggestion, will change.

Reply:
Why does this URL have 2 query strings?

Reply:
That is a typo! Fixing.

Reply:
Turns out the query string part is part of the hash and the `?` is what is specified in the [documentation](https://developers.google.com/analytics/devguides/config/provisioning/v3/user-controlled#technical-tos-flow-url).